### PR TITLE
Update features ibm_catalog.json

### PR DESCRIPTION
**Pull request by SirSpidey**
### Description

Makes the list of features parallel (start with verb).
Merges into PR #459 

### Release required?

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: ibm_catalog.json [hunk 1]
```diff
@@ -22,47 +22,42 @@
...
                 {
-                    "title": "Archiving",
+                    "title": "Add archive policies",
                     "description": "Add archive policies on the bucket."
                 },
                 {
-                    "title": "Activity Tracking",
-                    "description": "Integrate Activity Tracker events."
+                    "title": "Monitor Activity Tracker events",
+                    "description": "Integrate Activity Tracker events and provision IBM Cloud Monitoring to monitor how users interact with IBM Cloud Object Storage."
```
Comment (ocofaigh) [inline ibm_catalog.json:54]: The module does not support provisioning IBM Cloud Monitoring - it just integrates with it

File: ibm_catalog.json [hunk 2]
```diff
@@ -22,47 +22,42 @@
...
-                    "description": "Provision IBM Cloud Monitoring on Object Storage data."
-                },
-                {
-                    "title": "KMS encryption",
+                    "title": "Encrypt data with key management services",
                     "description": "Supports key management encryption (BYOK and KYOK), including service-to-service authorization policies with KMS."
-                },
                 {
-                    "title": "CBR",
+                    "title": "Secure data with CBR",
```
Comment (ocofaigh) [inline ibm_catalog.json:60]: I would say Secure access instead of data - the data itself is secured by encryption

**Review by ocofaigh — CHANGES_REQUESTED** [review]
Left some comment - Also Ill need to manually make these changes when finalised in the UI since the module is already onboarded and they wont get oicked up from the json again (sigh) (https://github.ibm.com/ibmcloud/content-catalog/issues/4329)

File: ibm_catalog.json (hunk 1 above)
Comment (SirSpidey) [reply ibm_catalog.json:54]: The old line 58 has "Provision IBM Cloud Monitoring on Object Storage data." So I guess that was wrong?

Comment (SirSpidey) [reply ibm_catalog.json:54]: Seems like a single feature about monitoring with AT and IBM Cloud monitoring makes sense though -- that we don't need 2 features listed.

**Comment by SirSpidey** [comment]
@ocofaigh pushed an update. Tried to fix the monitoring/AT feature.

File: ibm_catalog.json [hunk 3]
```diff
@@ -22,48 +22,44 @@
...
                 {
-                    "title": "Archiving",
+                    "title": "Add archive policies",
                     "description": "Add archive policies on the bucket."
                 },
                 {
-                    "title": "Activity Tracking",
-                    "description": "Integrate Activity Tracker events."
+                    "title": "Monitor activity",
+                    "description": "Integrate Activity Tracker events and IBM Cloud Monitoring to monitor how users interact with IBM Cloud Object Storage."
```
Comment (ocofaigh) [inline ibm_catalog.json:54]: Monitoring and activity tracker are two separate things, so maybe we need to split them into separate features:

```
"title": "Monitor data"
"description": "Integrate with the IBM Cloud Monitoring service to monitor your IBM Cloud Object Storage (COS) data in the IBM Cloud"
```
and
```
"title": "Activity tracking"
"Integrate with the IBM Cloud Activity Tracker service to track how users and applications interact with IBM Cloud Object Storage"
```

**Comment by SirSpidey** [comment]
anything else, @ocofaigh?

File: ibm_catalog.json (hunk 3 above)
Comment (SirSpidey) [reply ibm_catalog.json]: separated them. Check the descriptions

File: ibm_catalog.json [hunk 4]
```diff
@@ -22,48 +22,48 @@
...
-                    "title": "Activity Tracking",
-                    "description": "Integrate Activity Tracker events."
+                    "title": "Record Activity Tracker events",
+                    "description": "Integrate Activity Tracker to track events."
                 },
                 {
-                    "title": "Monitoring",
-                    "description": "Provision IBM Cloud Monitoring on Object Storage data."
+                  "title": "Monitor data",
+                  "description": "Provision IBM Cloud Monitoring on Object Storage data."
```
Comment (ocofaigh) [inline ibm_catalog.json:58]: This description is still wrong. It should say "Integrate with the IBM Cloud Monitoring service to monitor your IBM Cloud Object Storage data in the IBM Cloud"

**Comment by SirSpidey** [comment]
@ocofaigh if still not correct, just commit your change here and merge.

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**